### DISCUSSION TITLE
Commit Activities Not Showing in Activity Feed

### DISCUSSION THREAD

**Discussion by mrmamshad** [community/community#129453] [GitHub Education]
### Hi everyone,

Hello,

I am experiencing an issue with my GitHub account related to the activity feed. When I create a new repository, the creation event appears correctly in my activity feed. However, when I make commits to any of my repositories, these commit activities do not show up in the activity feed.

This discrepancy is causing some confusion, as it is important for me to track and display my ongoing development activities accurately. I have ensured that my commits are being pushed to the repositories without any errors, and I am not encountering any other issues with repository management or commit operations.

Could anyone provide guidance on how to resolve this issue?

Thank you for your assistance.

Best regards,  
@mrmamshad

**Comment by davevad93** [comment]
Hi @mrmamshad ,

There may be several reasons why your contributions are not showing up on your contributions graph:

1. You don't have the `Include private contributions on my profile` option checked in your `Settings`.
 ![1](https://github.com/community/community/assets/94308234/e0af8c1b-e9b5-45eb-ad0d-fb895534c4bf)

2. You have committed on a secondary `branch` of your repo, only commits made in the the repository's default branch are counted. If you merge the secondary `branch` into your default `branch` those commits plus the merged `pull request` will count as contributions.

4. You have made a commit with another GitHub account.

For more information about contributions not showing up in the profile, take a look at [this](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile) from the docs.
